extensions/pvaPublish/src/node/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ExtensionRegistration } from '@bfc/extension';
+
+import { getStatus, history, publish } from './publish';
+
+function initialize(registration: ExtensionRegistration) {
+  const extension = {
+    name: 'pva-publish-composer',
+    description: 'Publish bot to Power Virtual Agents (Preview)',
+    bundleId: 'publish',
+    history,
+    getStatus,
+    publish,
+    // TODO: add 'pull' once ready,
+  };
+
+  // @ts-ignore (TODO: remove once auth is integrated and added to publish method signature)
```

Comment:
```suggestion
  // @ts-expect-error (TODO: remove once auth is integrated and added to publish method signature)
```

This will start to fail the build when that condition is met.